lib/ClangImporter/ImportName.cpp

```diff
@@ -1427,6 +1427,12 @@ ImportedName NameImporter::importNameImpl(const clang::NamedDecl *D,
     case clang::OverloadedOperatorKind::OO_GreaterGreater:
     case clang::OverloadedOperatorKind::OO_AmpAmp:
     case clang::OverloadedOperatorKind::OO_PipePipe:
+    case clang::OverloadedOperatorKind::OO_Less:
+    case clang::OverloadedOperatorKind::OO_Greater:
+    case clang::OverloadedOperatorKind::OO_EqualEqual:
+    case clang::OverloadedOperatorKind::OO_ExclaimEqual:
+    case clang::OverloadedOperatorKind::OO_LessEqual:
+    case clang::OverloadedOperatorKind::OO_GreaterEqual:
```

Comment:
Can we keep this list (and also the test cases) in the same order as https://github.com/llvm/llvm-project/blob/master/clang/include/clang/Basic/OperatorKinds.def#L58,L104?

Reply:
Good idea. I did this as a separate commit.